core/src/test/java/org/apache/calcite/tools/PlannerTest.java

```diff
@@ -452,11 +452,6 @@ private void emptyUnions32(UnionMergeRule... extraRules)
         + "  EnumerableValues(tuples=[[{ 1 }]])\n"
         + "  EnumerableValues(tuples=[[{ 2 }]])\n";
 
-    checkUnionPruning("values(1)"
```

Comment:
what happened with this test?

Reply:
Originally this test would return:
`EnumerableValues(tuples=[[{ 1 }, { 2 }]])`
This is a little different from the plan above, it has to do with the order in which the Values are created, Values with Filters can't be merged, they are converted to unions. 
I'm not sure I need to add back this test.